shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-postgresql/src/main/java/org/apache/shardingsphere/data/pipeline/postgresql/ingest/PostgreSQLInventoryDumper.java

```diff
@@ -49,10 +52,26 @@ protected Object readValue(final ResultSet resultSet, final int index) throws SQ
         if (isPgMoneyType(resultSet, index)) {
             return resultSet.getBigDecimal(index);
         }
+        if (isPgBitType(resultSet, index)) {
```

Comment:
It's better to check sqlType (java.sql.Types) first (like `getObject` impl in PG driver), to reduce unnecessary string comparison.

Reply:
It could be on higher level, at `isPgMoneyType` and `isPgBitType` level. Refactor later.